viktorsha/Frontend_Tasks
# V1-03. Added routing, redirecting

You can check out the site by following [this link](https://viktorsha.github.io/Frontend_Tasks/).
### **Overview** ###
There's information about the site on the page. if you click on buttons on the top right corner, you'll be redirected to a specific page.
There are 4 routes: 
1. **"/my-notes"** for personal notes
2. **"/shared-notes"** for shared with other users notes
3. **"/about"** or **"/"** for information about the service
4. **"/not-found"** for other pages

### **Changes** ###
Added 4 routes and redirection to "/not-found" page, added styles for all pages.

### **How to test** ###
1. Open the page by following [this link](https://viktorsha.github.io/Frontend_Tasks/)
2. To test **personal notes**, click the _"My Notes" button_ on the top right corner.
3. To test **shared notes**, click the _"Shared Notes" button_ on the top right corner.
4. To test the **about page**, click the _"About" button_ on the top right corner.
5. To test the **not found page**, follow [this link](https://viktorsha.github.io/Frontend_Tasks/) and add any text after "Frontend_Tasks". You should be redirected to the "not-found" page.
